lib/curation_concerns/configuration.rb

```diff
@@ -127,6 +127,13 @@ def lock_retry_delay
       @lock_retry_delay ||= 200 # milliseconds
     end
 
+    # @!attribute [w] ingest_queue_name
+    #   ActiveJob queue to handle ingest-like jobs.
+    attr_writer :ingest_queue_name
+    def ingest_queue_name
```

Comment:
Given the discussion #788, I'd almost want to leave it as a "beta" setting until we are all happy with a pattern.